README.md

```diff
@@ -1,19 +1,79 @@
 
+# GitHub Workflow
 
+---
+### GitHub UI Instructions
+---
 
-# GitHub Workflow
+1. Fork this repository<br>
+   *Done from NSC GitHub Repository (click link below)*  
+   <a href="https://github.com/NSC-Computer-Science-Club/ForkPracticeCareerPrep" target="_blank">https://github.com/NSC-Computer-Science-Club/ForkPracticeCareerPrep</a>
+   ![Highlighting Fork Button Location](readme_images/fork.png)
+
+
+2. The repo will appear under your profile. Click this repository (We will do our work from here).
+
+   ![](readme_images/myProfileRepo.png)
+
+
+   > *For a Deeper dive Using Git Bash command line. Follow Steps **A** thru **G** Under **Terminal Commands*** below
+
+3. Click the green "Add file" button. Select "Create new file"
+
+   ![](readme_images/Adding%20file%20from%20UI.png)
+
+4. At the top of the file will be a textbox to name your file. Include the subfolder. Typing the name of the subfolder and then a forward-slash will indicate a new folder. Then type readme.md
+
+   ![](readme_images/newFile.png)
+   ![](readme_images/newFileWithFolder.png)
+
+5. Then add markdown of your choosing to illustrate what you learned. Here is a <a href="https://stackedit.io" target="_blank">web tool</a> to help with markdown.
+
+6. Once you are satisfied with your changes. Save your file by scrolling down and clicking the green "Commit new file" button.
+
+   ![](readme_images/commit.png)
+
+7. From your GitHub Profile submit a pull request
+
+   ![](readme_images/pullRequest.png)
+
+---
+### Terminal Commands
+---
+**Switch to the Git Bash terminal**
+
```

Comment:
Could the Terminal Commands section also be indented to match the GitHub UI section above, and to make it clearer/look nicer?